Tests/CommandsTests/PackageToolTests.swift

```diff
@@ -2953,4 +2953,117 @@ final class PackageToolTests: CommandsTestCase {
             XCTAssertNoDiagnostics(observability.diagnostics)
         }
     }
+    
+    // TODO: test archive signing
+    @available(macOS 12, iOS 15, tvOS 15, watchOS 8, *)
+    func testArchiveSigning() throws {
```

Comment:
@tomerd This needs to be updated. Right now it's just copy-paste of the publish test.